pkgs/applications/networking/browsers/elinks/default.nix

```diff
@@ -23,7 +24,15 @@ stdenv.mkDerivation {
     ./openssl-1.1.patch
   ];
 
-  buildInputs = [ ncurses xlibsWrapper bzip2 zlib openssl spidermonkey gpm ]
+  postPatch = (stdenv.lib.optional stdenv.isDarwin) ''
+    patch -p0 < ${fetchpatch {
+      url = "https://raw.githubusercontent.com/macports/macports-ports/72bed7749e76b9092ddd8d9fe2d8449c5afb1d71/www/elinks/files/patch-perl.diff";
+      sha256 = "14q9hk3kg2n2r5b062hvrladp7b4yzysvhq07903w9kpg4zdbyqh";
+    }}
```

Comment:
You can add the patch using `patches` and set `patchFlags = [ "-p0" ];` in the derivation.

Reply:
I've already tried to do it in such a way, it doesn't work for me:
```
patching sources
applying patch /nix/store/gfls3ypbdalhaihpdsfcpy5av4x3ambq-gc-init.patch
can't find file to patch at input line 5
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|Fix libgc initialization in the presence of Guile 2.0.
|
|--- elinks-0.12pre5/src/main/main.c    2009-07-07 14:23:17.000000000 +0200
|+++ elinks-0.12pre5/src/main/main.c    2011-04-28 23:20:15.000000000 +0200
--------------------------
File to patch:
Skip this patch? [y]
Skipping patch.
1 out of 1 hunk ignored
```

Reply:
Sorry, I've missed that there are multiple patches. LGTM then :+1: